Classes/Private/Manager/Audio/OCTAudioQueue.m

```diff
@@ -0,0 +1,418 @@
+OSStatus (*_AudioQueueDispose)(AudioQueueRef inAQ,
+                               Boolean inImmediate) = AudioQueueDispose;
+OSStatus (*_AudioQueueEnqueueBuffer)(AudioQueueRef inAQ,
+                                     AudioQueueBufferRef inBuffer,
+                                     UInt32 inNumPacketDescs,
+                                     const AudioStreamPacketDescription *inPacketDescs) = AudioQueueEnqueueBuffer;
+OSStatus (*_AudioQueueFreeBuffer)(AudioQueueRef inAQ,
+                                  AudioQueueBufferRef inBuffer) = AudioQueueFreeBuffer;
+OSStatus (*_AudioQueueNewInput)(const AudioStreamBasicDescription *inFormat,
+                                AudioQueueInputCallback inCallbackProc,
+                                void *inUserData,
+                                CFRunLoopRef inCallbackRunLoop,
+                                CFStringRef inCallbackRunLoopMode,
+                                UInt32 inFlags,
+                                AudioQueueRef *outAQ) = AudioQueueNewInput;
+OSStatus (*_AudioQueueNewOutput)(const AudioStreamBasicDescription *inFormat,
+                                 AudioQueueOutputCallback inCallbackProc,
+                                 void *inUserData,
+                                 CFRunLoopRef inCallbackRunLoop,
+                                 CFStringRef inCallbackRunLoopMode,
+                                 UInt32 inFlags,
+                                 AudioQueueRef *outAQ) = AudioQueueNewOutput;
+OSStatus (*_AudioQueueSetProperty)(AudioQueueRef inAQ,
+                                   AudioQueuePropertyID inID,
+                                   const void *inData,
+                                   UInt32 inDataSize) = AudioQueueSetProperty;
+OSStatus (*_AudioQueueStart)(AudioQueueRef inAQ,
+                             const AudioTimeStamp *inStartTime) = AudioQueueStart;
+OSStatus (*_AudioQueueStop)(AudioQueueRef inAQ,
+                            Boolean inImmediate) = AudioQueueStop;
+#if ! TARGET_OS_IPHONE
+OSStatus (*_AudioObjectGetPropertyData)(AudioObjectID inObjectID,
+                                        const AudioObjectPropertyAddress *inAddress,
+                                        UInt32 inQualifierDataSize,
+                                        const void *inQualifierData,
+                                        UInt32 *ioDataSize,
+                                        void *outData) = AudioObjectGetPropertyData;
+#endif
+
+static NSError *OCTErrorFromCoreAudioCode(OSStatus resultCode)
+{
+    return [NSError errorWithDomain:NSOSStatusErrorDomain
+                               code:resultCode
+                           userInfo:@{NSLocalizedDescriptionKey : @"Consult the CoreAudio header files/google for the meaning of the error code."}];
+}
+
+#if ! TARGET_OS_IPHONE
+static NSString *OCTGetSystemAudioDevice(AudioObjectPropertySelector sel, NSError **err)
```

Comment:
No, I mean those functions don't need the self parameter. They're not conceptually related to the class so I moved them out.